BitwardenShared/Core/Platform/Services/AuthenticatorSyncService.swift

```diff
@@ -160,16 +160,16 @@ actor DefaultAuthenticatorSyncService: NSObject, AuthenticatorSyncService {
         let decryptedCiphers = try await totpCiphers.asyncMap { cipher in
             try await self.clientService.vault(for: userId).ciphers().decrypt(cipher: cipher)
         }
-        let account = try await stateService.getActiveAccount()
-        let username = account.profile.email
+        let account = try? await stateService.getAccount(userId: userId)
 
         return decryptedCiphers.map { cipher in
             AuthenticatorBridgeItemDataView(
+                bitwardenAccountName: account?.profile.email,
```

Comment:
Currently, the comps for the Authenticator don't distinguish organizations or collections. It's just one big list under the account. This field is just to support the items we need to show over there. Maybe we should call it `bitwardenAccountEmail` since it will be the email that we're displaying?

In addition, your question here led me to clarify one other UX question I had and it sounds like we _do_ need to store the domain. I am going to add that as a column now.

Reply:
I renamed it to `bitwardenAccountEmail` for clarity and added `bitwardenAccountDomain` in the latest commit 👍